api/v1/standalone-dns-proxy/standalone-dns-proxy.proto

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+syntax = "proto3";
+
+package standalonednsproxy;
+
+option go_package = "github.com/cilium/cilium/api/v1/standalonednsproxy";
+
+service FQDNData {
+  rpc SubscribeToDNSPolicies(stream  DNSPoliciesResult) returns (stream DNSPolicies) {}
+  rpc UpdatesMappings(FQDNMapping) returns (UpdatesMappingsResult) {}
+}
+
+message DNSPoliciesResult {
+    bool success = 1;
```

Comment:
Can we make this an enum instead of a bool, to be able to convey more than one bit of information? This may well be a bit pointless at the moment, but I'm inclined to believe this will change in the future. (c.f. https://katafrakt.me/2024/11/09/booleans-are-a-trap/)